src/App.jsx

```diff
@@ -4,11 +4,14 @@ import Home from "./pages/home.jsx"
 import Symptome from "./pages/symptome.jsx"
 import Medikamente from "./pages/medikamente.jsx"
 import Termine from "./pages/termine.jsx"
+import NeuerTermin from "./pages/neuertermin.jsx"
+import NeuesMedikament from "./pages/neuesmedikament.jsx"
 import Footer from './components/Footer/Footer.jsx'
 
 
 
 
+
```

Comment:
`npm i --save-dev prettier`
`client $ npx prettier --write .` (save files)
Save a few bytes and the environment.

Reply:
Done!